packages/xo-server/src/api/backup.js

```diff
@@ -62,7 +65,7 @@ function handleFetchFiles (
 
       const archive = archiver(archiveFormat)
       archive.on('error', error => {
-        console.error(error)
+        log.error(error)
```

Comment:
Incorrect use, missing `message` param.

Should we add support for this in `@xo/log`?